transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUring.java

```diff
@@ -41,7 +43,8 @@ public final class IoUring {
                     try {
                         ringBuffer = Native.createRingBuffer();
                         Native.checkAllIOSupported(ringBuffer.fd());
-                        supported = Native.isIOUringCqeFSockNonEmptySupported(ringBuffer.fd());
+                        socketNonEmptySupported = Native.isIOUringCqeFSockNonEmptySupported(ringBuffer.fd());
+                        spliceSupported = Native.isIOUringCqeFSockNonEmptySupported(ringBuffer.fd());
```

Comment:
replace with `Native.isIOUringSpliceSupported`

Reply:
> replace with 'isIOUringSpliceSupported`

I get it. Here is a mistake of mine.